docs/documentation/maya/edge_evaluator.md

```diff
@@ -1,34 +1,29 @@
-# How To Use
+# AdnEdgeEvaluator
 
-The Edge Evaluator Node is a dependency node to compute deformation changes in edges. Based on two input meshes will output a compression map of the edges.
+The AdnEdgeEvaluator node is a dependency node that computes deformation changes in edges. Based on two input meshes it will output a compression map of the edges.

```

Comment:
We should maybe add an image with the result of the edhe evaluator and maybe an example of how to use or connect it to out deformers

Reply:
I don't have the resources at the moment to provide that screenshot. I'm approving this PR given its current status